src/index.ts

```diff
@@ -82,13 +91,21 @@ const defaultLogger = (options: CosmasOptions & { loggerName?: string } = {}): C
     const messageKey = 'message'; // best option for Google Stackdriver,
     const streams = initLoggerStreams(defaultLevel, Object.assign({}, options, { messageKey }));
 
-    options.ignoredHttpMethods = options.ignoredHttpMethods || ['OPTIONS'];
+    const formatters: { level?: (label: string, level: number) => object } = {};
```

Comment:
Maybe we can use `pino.LoggerOptions['formatters']` from pino?

Reply:
Fixed in c772474e33fb04625ca748dbbd747474842e223f